hyperspy/signal.py

```diff
@@ -741,10 +758,11 @@ def plot_decomposition_factors(self,
                                    comp_ids,
                                    calibrate=True,
                                    same_window=None,
-                                   comp_label='Decomposition factor',
```

Comment:
yes... I will leave both and put a `DeprecationWarning` for this version? Should I then make a PR to next_release_major to remove both the `comp_label` argument and the `DeprecationWarning`?